src/client/sandbox/node/window.js

```diff
@@ -420,6 +420,28 @@ export default class WindowSandbox extends SandboxBase {
             };
         }
 
+        if (window.WebSocket) {
+            window.WebSocket = function (url, protocols) {
+                if (arguments.length === 0)
```

Comment:
`WebSocket.url` should be proxied.
I guess you can override property value descriptor here for `window.WebSocket.prototype.url`.